src/software_development/installation.rst

```diff
@@ -63,14 +63,20 @@ The following steps should be taken to install the real-time kernel:
 
   .. code-block:: bash
 
-    KERNEL_RT_VERSION="5.14.0-362.13.1.el9_3" && dnf install -y kernel-rt-${KERNEL_RT_VERSION} kernel-rt-devel-${KERNEL_RT_VERSION}
+    KERNEL_RT_VERSION="5.14.0-362.13.1.el9_3" && sudo dnf install -y kernel-rt-${KERNEL_RT_VERSION} kernel-rt-devel-${KERNEL_RT_VERSION}
 
 2. Add users to the **realtime** group, to allow access to the real-time kernel, for example:
 
   .. code-block:: bash
 
     sudo usermod -a -G realtime gmto
 
+3. Set the real-time kernel as the default kernel
+
+  .. code-block:: bash
+
+    sudo grubby --set-default /boot/vmlinuz-${KERNEL_RT_VERSION}.x86_64+rt
```

Comment:
If we do it here, we might remove it from [etherlab_install](https://github.com/GMTO/ocs_etherlabmaster_ext/blob/master/etherlab_install.sh#L29) script which also calls grubby.

Reply:
yes, but you may want to use the real-time kernel but not the ethercat framework